app/uk/gov/hmrc/personaldetailsvalidation/matching/MatchingConnector.scala

```diff
@@ -64,12 +64,10 @@ class FuturedMatchingConnector @Inject()(httpClient: HttpClient, connectorConfig
   }
 
   private implicit class PersonalDetailsSerializer(personalDetails: PersonalDetails) {
-    lazy val toJson: JsObject = Json.obj(
-      "firstName" -> personalDetails.firstName,
-      "lastName" -> personalDetails.lastName,
-      "dateOfBirth" -> personalDetails.dateOfBirth,
-      "nino" -> personalDetails.nino
-    )
+    lazy val toJson: JsObject = {
```

Comment:
We choose not to use Json.format because we want to control what data we are sending out to the authenticator. This future-proofs us by not sending additional fields (which we can add in future) in PersonalDetails case class.